### ISSUE TITLE
IOS - Upload doesn't work

### ISSUE THREAD

**Issue by dangllucas** [vaadin/framework#10409]
Simple UI with just a upload Button to upload a Image and receive its contents.

On Android, Windows or Linux everything works as expected.
File Chooser opens and i receive file on server side vaadin.

But on IOS 11 (Iphone 7s) after I pressed the Upload button just nothing happens.

Vaadin Framework Version: 8.1.7

**Comment by dangllucas** [comment]
Ok, got some News here.

I tried many different things. First thing was to change Html of input element with JavaScript, to only allow images.
Nothing helped and happened. So I changed it back.

Then I accidentally missclicked right under the Upload Button and suddenly a Window opens, that let me upload a Image.
I don't know if this is a Bug in IOS or in SAFARI or in VAADIN. 

But maybe it depends on Screen Height or other stuff. 

Hope you have an idea how to fix this.
